tests/virt_control_plane_test.go

```diff
@@ -165,39 +165,61 @@ var _ = Describe("[Serial][ref_id:2717][sig-compute]KubeVirt control plane resil
 		})
 
 		When("evicting pods of control plane", func() {
-			test := func(podName string) {
+			evictAllPods := func(podName string) error {
 				By(fmt.Sprintf("Try to evict all pods %s\n", podName))
 				podList, err := getPodList()
 				Expect(err).ToNot(HaveOccurred())
 				runningPods := getRunningReadyPods(podList, []string{podName})
 				Expect(len(runningPods)).ToNot(Equal(0))
 				for index, pod := range runningPods {
 					err = virtCli.CoreV1().Pods(flags.KubeVirtInstallNamespace).Evict(context.Background(), &v1beta1.Eviction{ObjectMeta: metav1.ObjectMeta{Name: pod.Name}})
-					// trying to evict the last running pod in this list should fail
 					if index == len(runningPods)-1 {
-						Expect(err).To(HaveOccurred(), "no error occurred on evict of last pod")
+						return err
 					} else {
 						Expect(err).ToNot(HaveOccurred())
 					}
 				}
+				return nil
 			}
 
-			It("[test_id:2830]last eviction should fail for virt-controller pods", func() { test("virt-controller") })
-			It("[test_id:2799]last eviction should fail for virt-api pods", func() { test("virt-api") })
+			It("[test_id:2830]last eviction should fail for multi-replica virt-controller pods", func() {
+				tests.SkipIfSingleReplica(virtCli)
+				// trying to evict the last running pod should fail
+				err := evictAllPods("virt-controller")
+				Expect(err).To(HaveOccurred(), "no error occurred on evict of last virt-controller pod")
+			})
+			It("[test_id:2799]last eviction should fail for multi-replica virt-api pods", func() {
+				tests.SkipIfSingleReplica(virtCli)
+				// trying to evict the last running pod should fail
+				err := evictAllPods("virt-api")
+				Expect(err).To(HaveOccurred(), "no error occurred on evict of last virt-api pod")
+			})
+			It("eviction of single-replica virt-controller pod should succeed", func() {
+				tests.SkipIfMultiReplica(virtCli)
+				err := evictAllPods("virt-controller")
+				Expect(err).ToNot(HaveOccurred(), "error occurred on eviction of single-replica virt-controller pod")
+			})
+			It("eviction of multi-replica virt-api pod should succeed", func() {
```

Comment:
Nit: you could easily make a table out of these tests.

Reply:
Done